### PR TITLE
Move log message output(fprintf, puthcar) to target code

### PR DESCRIPTION
**Pull request by seanshpark**
Related issue #752

JerryScript-DCO-1.0-Signed-off-by: SaeHie Park saehie.park@samsung.com

### CODE REVIEW

File: main-darwin.cpp [hunk 1]
```diff
@@ -317,3 +317,38 @@ main (int argc,
     }
   }
 }
+
+/**
+ * Provide log message to filestream implementation for the engine.
+ */
+int jerry_port_logmsg (FILE* stream, const char* format, ...)
```
Comment (egavrin) [inline main-darwin.cpp]: We can put `jerry_port_logmsg`, `jerry_port_errormsg` and `jerry_port_putchar` to `main.h` until implementation of these function would become different.

**Comment by seanshpark** [comment]
@egavrin , moved `jerry_port_l@ogmsg()`, `jerry_port_errormsg()` and `jerry_port_putchar()` to `main.h` file as your comment.

**Comment by egavrin** [comment]
LGTM

**Comment by seanshpark** [comment]
@LaszloLango , any comments?

**Comment by LaszloLango** [comment]
Sorry for the long delay. Looks good to me.

**Comment by seanshpark** [comment]
Thank you :)

**Pull request merged**